### ISSUE TITLE
MIB definition for trap-oid-prefix.1.1.[1..9]

### ISSUE THREAD

**Issue by amarbs** [SUSE/prometheus-webhook-snmp#33]
This is a very unique project and useful for integration with legacy systems.

Is there a MIB defined for the OIDs `trap-oid-prefix.1.1.[1..9]`? If its already present, it would be easy to import it into our custom enterprise traps MIB.

**Comment by votdev** [comment]
Yes, have a look [here](https://github.com/SUSE/prometheus-webhook-snmp/blob/master/PROMETHEUS-ALERT-CEPH-MIB.txt).
